### ISSUE TITLE
[Bug]: Translucent Water Bucket

### ISSUE THREAD

**Issue by cech12** [cech12/BucketLib#19]
### Minecraft Version

1.19

### Forge Version

41.0.76+

### Mod Version

1.19-1.0.1.0

### Describe the Issue

You can see through a water bucket generated by this library. The update (https://github.com/cech12/BucketLib/issues/18) after the breaking change of Forge 41.0.64 caused this issue. It should be fixed. :) 

Interesting links:
- https://github.com/MinecraftForge/MinecraftForge/commit/ce3d8b40cf37924caf1708cdde6842ae6fdcee31

```
{
  "parent": "forge:item/bucket",
  "fluid": "minecraft:water",
  "loader": "forge:fluid_container"
}
```

Try if Forge 41.0.95 (8854) could fix it. - it doesn't

**Comment by cech12** [comment]
Opened Forge issue https://github.com/MinecraftForge/MinecraftForge/issues/8871

**Comment by cech12** [comment]
Updating to Forge 41.0.105 should fix the issue

**Comment by cech12** [comment]
Fixed and released in 1.19-1.0.3.0
